server/dockerStart.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import getopt
+import subprocess
+import os
+
+
+def printHelp():
+    print("USAGE: dockerStart.py [OPTIONS]\n" +
+          "  Running dockerStart.py with no options is equivalent to:\n" +
+          "  dockerStart.py -b --port=5000 --url=0.0.0.0\n\nOptions:" +
+          "     -h, --help       Show this command list\n" +
+          "(Turns on debug mode for Flask, and installs pytest)\n" +
+          "     -p, --prod       For production (Enables nginx)\n" +

```

Comment:
Docker should be prod only. We do not need to specify prod or not

Reply:
I removed the dev option and removed this script because the user will not need to specify a environment.